modular_skyrat/master_files/code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -0,0 +1,29 @@
+//Used for Abductors
+/datum/reagent/medicine/abductoradrenaline
+	name = "Z-Gamma Precepts"
+	description = "A synthesized adrenaline based on a chemical a certain other lifeform creates. Overdosing can cause brain damage."
+	color = "#4CFF00"
+	overdose_threshold = 30
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/medicine/changelingadrenaline/on_mob_life(mob/living/carbon/metabolizer, delta_time, times_fired)
```

Comment:
not sure if you meant for this to be changelingadrenaline or abductoradrenaline

Reply:
fixed